package.json

```diff
@@ -82,7 +85,8 @@
     "typedoc": "^0.16.0",
     "typedoc-plugin-external-module-map": "^1.2.1",
     "typescript": "^3.8.0",
-    "uuid": "^3.3.2"
+    "uuid": "^3.3.2",
+    "which": "2.0.2"
```

Comment:
If "which" is only used by integration tests inside "packages/cli", should the packages "which" and "@types/which" be dev dependencies in packages/cli/package.json instead of here at the top level?

Reply:
yep.